src/App.jsx

```diff
@@ -143,12 +143,19 @@ function App() {
       // then user DOES have a wallet
       connect().then(() => {
         setWalletChecked(true);
+        const providerURL = uri;
+        // Note (appleseed): remove this before merge to develop
+        console.log("segment debug", providerURL, currentPath);
+        segmentUA({
+          type: "connect",
+          provider: provider,
+          context: currentPath,
+        });
```

Comment:
just pulling uri from web3Context. uri is importable in all views now if you need it elsewhere...

Reply:
@casuwu I left that console.log in there in case you wanted to test the implementation. If you're good with it just remove that console.log, then let's merge it up